### ISSUE TITLE
Add to CDN

### ISSUE THREAD

**Issue by jenil** [rsms/interface#6]
hey! 

First, great work here!   👏🏻

I was referencing `https://rsms.me/interface/interface.css` but the server seems to be slow and wasn't even resolving today morning. Do you mind adding it to a CDN provider like cdnjs.com or even a npm package that can be loaded over unpkg.com?

**Comment by MartinKolarik** [comment]
@jenil You can also use jsDelivr CDN for production, which can serve all files from GitHub: https://cdn.jsdelivr.net/gh/rsms/interface@1.1/

**Comment by jenil** [comment]
I see, yeah I saw it was behind a Cloudflare CDN and that totally makes sense. 

I'd still want you to consider publishing it on `npm` for an easier availability for a lot of other projects.

Thanks!

**Comment by rsms** [comment]
<img width="989" alt="screenshot 2017-08-23 08 36 04" src="https://user-images.githubusercontent.com/47046/29624580-4d6e81f0-87de-11e7-8dac-72b6abc1fcce.png">

**Comment by rsms** [comment]
@MartinKolarik Yeah, you could do that — put a copy of interface.css into your web thing and point to font files at https://cdn.jsdelivr.net/gh/rsms/interface@1.1/docs/font-files/

However, note that rsms.me is itself served from Github pages.

**Comment by MartinKolarik** [comment]
Ah, I wanted to say the CSS file is also [available here](https://cdn.jsdelivr.net/gh/rsms/interface@1.1/docs/interface.css) but then I realized it uses absolute URLs. I'd recommend changing that to relative ones in the future, to make it easier to use.

**Comment by terinjokes** [comment]
@rsms :grimacing: You probably want to configure a Page Rule to cache the CSS file and font-face so they remain in Cloudflare's cache for longer.
